frontend/src/components/__tests__/Header.test.ts

```
@@ -189,8 +188,8 @@ describe('Header', () => {
       setAuth(isAuth, username, role);
       await openMobileMenu();
       await waitFor(() => {
-        const mobileMenu = document.body.querySelector('.lg\\:hidden.absolute');
-        expectedContent.forEach(text => expect(mobileMenu?.textContent).toContain(text));
+        const mobileMenu = screen.getByTestId('mobile-menu');
+        expectedContent.forEach(text => expect(mobileMenu.textContent).toContain(text));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)